src/util/wallet-utils/src/getAddress.ts

```diff
@@ -99,10 +106,15 @@ export const getAddressesFordefi: getAddresses = async (params) => {
 };
 
 export const getAddressesAsigna: getAddresses = async (params) => {
+  return Promise.race([getBTCAddressAsigna(params), getStxAddressAsigna()]);
```

Comment:
IIUC `race` will resolve with the first resolution, being it a success or failure. Is this what we want? vs using `any`

Reply:
good idea asigna might change their apis in the future any would future proof this